### ISSUE TITLE
/ by 0 exception when first number is 0

### ISSUE THREAD

**Issue by msassarini** [msassarini/fxMCD#4]
Example 0, 10, 1 

The not 0 check is not done on the first number of the set.

**Comment by msassarini** [comment]
Note to self:

### Function MCD
```
if (num1 == 0) {
  return num2;
}
```
### Function mcm
```
if (num1 == 0 || num2 == 0) {
  return 0;
}
```

**Comment by msassarini** [comment]
Bug fixed in new release #5
